BREAKING.md

```diff
@@ -13,13 +13,18 @@ There are a few steps you can take to write a good change note and avoid needing
 ## 0.56 Breaking changes
 - [`MessageType.Save` and code that handled it was removed](#messageType-save-and-code-that-handled-it-was-removed)
 - [Removed `IOdspResolvedUrl.sharingLinkToRedeem`](#Removed-IOdspResolvedUrl.sharingLinkToRedeem)
+- [`wait()` methods removed from map and directory](#wait()-methods-removed-from-map-and-directory)
 
 ### `MessageType.Save` and code that handled it was removed
 The `Save` operation type was deprecated and has now been removed. This removes `MessageType.Save` from `protocol-definitions`, `save;${string}: ${string}` from `SummarizeReason` in the `container-runtime` package, and `MessageFactory.createSave()` from and `server-test-utils`.
 
 ### Removed `IOdspResolvedUrl.sharingLinkToRedeem`
 The `sharingLinkToRedeem` property is removed from the `IOdspResolvedUrl` interface. The property can be accesed from `IOdspResolvedUrl.shareLinkInfo` instead.
 
+### `wait()` methods removed from map and directory
```

Comment:
```suggestion
### wait() methods removed from map and directory
```
trying no more inline code blocks in headers and reducing them in descriptions (n/a here because it's short enough) (also ignore the existing two entries for this release that made it in before we decided this)

Reply:
Fixed in latest